README.md

```diff
@@ -10,6 +10,7 @@ LightGBM, Light Gradient Boosting Machine
 [![PyPI Version](https://img.shields.io/pypi/v/lightgbm.svg?logo=pypi&logoColor=white)](https://pypi.org/project/lightgbm)
 [![Join Gitter at https://gitter.im/Microsoft/LightGBM](https://badges.gitter.im/Microsoft/LightGBM.svg)](https://gitter.im/Microsoft/LightGBM?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Slack](https://lightgbm-slack-autojoin.herokuapp.com/badge.svg)](https://lightgbm-slack-autojoin.herokuapp.com)
+[![Optuna](https://img.shields.io/badge/Optuna-integrated-blue)](https://medium.com/optuna/lightgbm-tuner-new-optuna-integration-for-hyperparameter-optimization-8b7095e99258)
```

Comment:
@Crissman Thanks a lot for your PR!

I don't think that this is the right place to give links for a third-party software. We use badges only for technical information related to LightGBM directly.

We have a special section where we place links to third-party software: https://github.com/microsoft/LightGBM#external-unofficial-repositories. Please move the link to Optuna there and follow the existing list style: `Name[ (description)]: GitHub_repo_url`.

Reply:
Thanks, @StrikerRUS . Removed the badge. Please merge!

Reply:
@Crissman Can you please add Optuna to https://github.com/microsoft/LightGBM#external-unofficial-repositories?

Reply:
@StrikerRUS, the current link is not to a repository, but rather the blog post that explains how to use the Optuna LightGBM Tuner. Did you want me to add an additional line for the Optuna Github repository to the External Unofficial Repositories section?

Thanks,

Reply:
@Crissman 
> Did you want me to add an additional line for the Optuna Github repository to the External Unofficial Repositories section?

Yep, exactly!

Reply:
Done! Please take a look.

Reply:
Thanks!